ogr/ogrsf_frmts/CMakeLists.txt

```diff
@@ -96,6 +96,8 @@ ogr_dependent_driver(cad "OpenCAD" "GDAL_USE_OPENCAD OR GDAL_USE_OPENCAD_INTERNA
 ogr_dependent_driver(parquet "Parquet" "GDAL_USE_PARQUET")
 ogr_dependent_driver(arrow "Arrow" "GDAL_USE_ARROW")
 
+ogr_dependent_driver(gtfs "GTFS" "OGR_ENABLE_DRIVER_CSV")
```

Comment:
Shouldn't it be OGR_ENABLE_DRIVER_GTFS ?

Reply:
no, the GTFS driver requires (at runtime) the CSV driver